### PR TITLE
added new js

### PR DESCRIPTION
**Pull request by panncho453**

### CODE REVIEW

File: assets/park.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+function getparkcode() {
+Storage.getitem(parkcode)
```
Comment (CECurtiss) [inline assets/park.js:2]: Delete line 2

File: assets/park.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+    var parkcodesearch = storage.getitem(parkcode)
```
Comment (CECurtiss) [inline assets/park.js:3]: camel case Item in getItem

**Review by CECurtiss — DISMISSED** [review]
edit a few lines

**Review by DeadEye44 — APPROVED** [review]

**Comment by DeadEye44** [comment]
Will need to create a new pull request for this and make all changes before pushing it.

**Review by DeadEye44 — APPROVED** [review]

**Pull request merged**